cmd/logs.go

```diff
@@ -91,7 +112,9 @@ func newSchedulerLogsCmd() *cobra.Command {
 astro deployment logs scheduler YOU_DEPLOYMENT_ID -s string-to-find
 `,
 		Args: cobra.ExactArgs(1),
-		RunE: schedulerRemoteLogs,
+		RunE: func(cmd *cobra.Command, args []string) error {
+			return fetchRemoteLogs("scheduler", args)
```

Comment:
See my previous comment, we should use constants for `scheduler`

Reply:
moved to a constant